spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -102,19 +102,21 @@ def get_parser():
             "template label. No scaling will be performed. \n"
             "\n"
             "If two labels are provided, a linear transformation (translation + rotation + superior-inferior linear "
-            "scaling) will be applied. The strategy here is to defined labels that cover the region of interest. For "
+            "scaling) will be applied. The strategy here is to define labels that cover the region of interest. For "
             "example, if you are interested in studying C2 to C6 levels, then provide one label at C2 and another at "
             "C6. However, note that if the two labels are very far apart (e.g. C2 and T12), there might be a "
             "mis-alignment of discs because a subject''s intervertebral discs distance might differ from that of the "
             "template.\n"
             "\n"
-            "If more than two labels (only with the parameter '-disc') are used, a non-linear registration will be "
-            "applied to align the each intervertebral disc between the subject and the template, as described in "
-            "sct_straighten_spinalcord. This the most accurate and preferred method. This feature does not work with "
-            "the parameter '-ref subject', where only a rigid registration is performed.\n"
+            "If more than two labels are used, a non-linear registration will be applied to align the each "
+            "intervertebral disc between the subject and the template, as described in "
+            "sct_straighten_spinalcord. This the most accurate and preferred method. Note: \n"
```

Comment:
```suggestion
            "sct_straighten_spinalcord. Note: \n"
```
i don't think we want to endorse this as the best method-- there are serious cons associated with it